client/comms/server.js

```diff
@@ -18,6 +18,26 @@ var Client = function (id_, conn_) {
   this.isTeleporting = false;
   this.loadoutIDX = 0;
   this.loadout = new Loadout(Loadout.SCOUT);
+  //TODO: Remove the advanced loadouts when we have a way to earn the in-game
+  this.unlockedLoadouts = [
```

Comment:
maybe this should be a list of instantiated loadout objects that each store the number of shots left

Reply:
Updated